src/Elcodi/Admin/MediaBundle/Resources/views/Module/_images-form-field.html.twig

```diff
@@ -32,7 +32,10 @@
                     </a>
                     <ul class="thumbnail-actions">
                         <li>
-                            <a href="{{ url('admin_image_delete', { id: image.id }) }}" class="icon-trash-o"></a>
+                            <a
+                                href="{{ url('admin_image_delete', { id: image.id }) }}{% if redirect_url is defined %}{{ '?redirect-path=' ~ redirect_url }}{% endif %}"
```

Comment:
This asumes knowledge about how the url will be generated.
I'd take advantage of passing extra parameters to a route, since they're added like a querystring, so this will become:
```
url('admin_image_delete, {
    id: image.id,
    redirect_url: redirect_url|default(null)
})
```